Project/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -6,6 +6,6 @@ EditorBuildSettings:
   serializedVersion: 2
   m_Scenes:
   - enabled: 1
-    path: Packages/com.unity.ml-agents/Tests/Runtime/SerializeTestScene.unity
-    guid: 60783bd849bd242eeb66243542762b23
+    path: Assets/ML-Agents/Examples/3DBall/Scenes/3DBall.unity
```

Comment:
Are you intending to change this for the default build settings?

Reply:
Damn, I did the same thing you did. I think we need to remove SerializeTestScene from the settings, but I'll do that separately.

Reply:
Reverted.